.gitignore

```diff
@@ -188,6 +188,7 @@ subscriberAgentWithJ9threadTrace2.out
 tfileTest*.tst
 traceLogTest.trc
 VerboseGC_*.xml*
+*.ilt
```

Comment:
What's an "ilt" file?

Reply:
Those were the files generated by the recorder: ilt -> intermediate language text. I'll remove it.